src/shared/components/Resources/ResourceList.tsx

```diff
@@ -1,10 +1,16 @@
 import * as React from 'react';
-import { List } from 'antd';
+import { List, Drawer } from 'antd';
 import { PaginatedList, Resource, PaginationSettings } from '@bbp/nexus-sdk';
 import ResourceItem, { ResourceItemProps } from './ResourceItem';
-
 import './Resources.less';
 
+let ReactJson: any;
+if (typeof window !== 'undefined') {
+  ReactJson = require('react-json-view').default;
+}
+
+console.log(ReactJson);
```

Comment:
Looks like this should go away

Reply:
I keep leaving these here maybe I should lint them out... 😅

Reply:
I tried to configure the linter but it seems the esLint "overrides" object doesn't work for us to target only client-side files for console linting. A shame!